### PR TITLE
Add postgresql page to data bubble and subnav

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built page as per [doc](https://docs.google.com/document/d/1GxineF4WLt0YlShaYkrzqWjIOkXjLJEfsnUdWVfWnz4/edit) and [design](https://www.figma.com/file/17XOOU2kMgAW7CsRhGyu4Y/23.10-C.com---Data-Fabric-bubble-(rebranding)?node-id=128%3A1198&mode=dev)

## QA

- View the site locally in your web browser at: https://canonical-com-1124.demos.haus/data/postgresql
- Check against doc and design on all screen sizes 

## Issue / Card

Fixes # https://warthogs.atlassian.net/browse/WD-6433

### CODE REVIEW

**Comment by mtruj013** [comment]
Blocked pending missing links.

**Comment by carkod** [comment]
@lyubomir-popov can you review?

**Comment by juanruitina** [comment]
- [ ] Please put the Azure logo between AWS and Google Cloud as per the copy doc.
- [ ] PostgreSQL logo looks blurry on my screen (I know we've been having some issues with this lately, I think there's some kind of flag)
Screenshot 2023-11-29 at 15 15 37

- [ ] The links in the resource section seem to lack some top margin/padding. Wrapping them in an `p` element seems to do the trick, but I don't know if that's the best practice.

Screenshot 2023-11-29 at 15 21 10

Note: in the design all the logos are on a single row, maybe something to look at, having one or two "orphan" logos isn't great.

Feel free to change the label to UX+1 once you fix the first point, the other bits are visual anyway.

**Comment by mtruj013** [comment]
@juanruitina I reordered the logos that way as given of how many there are and how wide some of them are, there will always be at least 1 orphan. So there are basically 3 options here:

1. I order based on the doc without adding a break and have single orphan (the design feedback I've received in the past is that single orphans are not acceptable)
Screenshot from 2023-11-29 15-55-27

2. I order based on the doc WITH a break and have weird spacing on the right most side of the first row:
image

4. I keep it as is:
image


Let me know what you prefer and I'll make the necessary changes

**Comment by lyubomir-popov** [comment]
Pushed a tiny fix, looks good, +1 @mtruj013

**Comment by lyubomir-popov** [comment]
2 works best IMO

**Comment by lyubomir-popov** [comment]
@mtruj013 for the logo section, version 2 please.

**Comment by mtruj013** [comment]
> for the logo section, version 2 please.

Already done :)

File: templates/data/postgresql.html [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+        </div>
+        <div class="col-4"> 
+          <a href="https://ubuntu.com/engage/database-cloud-migration">Database cloud migration</a>
+          <p>Best practices to migrate your data to the cloud.</p>
+          <hr class="p-rule"/>
+        </div>
+        <div class="col-4 col-start-large-3"> 
+          <a href="https://ubuntu.com/engage/database-cloud-migration">Database cloud migration</a>
+          <p>Best practices to migrate your data to the cloud.</p>
+        </div>
```
Comment (aaryanporwal) [inline templates/data/postgresql.html:437]: Duplicate links

**Review by aaryanporwal — CHANGES_REQUESTED** [review]
Everything looks good :) Just one comment below.

**Review by aaryanporwal — APPROVED** [review]
LGTM

**Pull request merged**